composer.json

```
@@ -7,18 +7,18 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "silverstripe/recipe-plugin": "^1.5",
-        "silverstripe/recipe-core": "4.x-dev",
-        "silverstripe/admin": "1.x-dev",
-        "silverstripe/asset-admin": "1.x-dev",
-        "silverstripe/campaign-admin": "1.x-dev",
-        "silverstripe/versioned-admin": "1.x-dev",
-        "silverstripe/cms": "4.x-dev",
-        "silverstripe/errorpage": "1.x-dev",
-        "silverstripe/reports": "4.x-dev",
-        "silverstripe/siteconfig": "4.x-dev",
-        "silverstripe/versioned": "1.x-dev",
-        "silverstripe/graphql": "3.x-dev || 4.x-dev",
-        "silverstripe/session-manager": "1.x-dev"
+        "silverstripe/recipe-core": "4.11.x-dev",
+        "silverstripe/admin": "1.11.x-dev",
+        "silverstripe/asset-admin": "1.11.x-dev",
+        "silverstripe/campaign-admin": "1.11.x-dev",
+        "silverstripe/versioned-admin": "1.11.x-dev",
+        "silverstripe/cms": "4.11.x-dev",
+        "silverstripe/errorpage": "1.11.x-dev",
+        "silverstripe/reports": "4.11.x-dev",
+        "silverstripe/siteconfig": "4.11.x-dev",
+        "silverstripe/versioned": "1.11.x-dev",
+        "silverstripe/graphql": "4.0.x-dev",
```

```suggestion
        "silverstripe/graphql": "3.8.x-dev  || 4.0.x-dev",
```
We're still supporting both versions of graphql

---

We do, however in order to use graphql 3 you need to 'inline' (stop using) recipe-cms

---

Ahh yup, good point. I do remember seeing that in the doc.